scripts/inactive_study.py

```diff
@@ -162,3 +174,60 @@
     plt.legend(loc='lower right')
     plt.savefig('inactive_study/' + args.script_name + '_k' + str(nl) + '.pdf', bbox_inches='tight')
     plt.close()
+
+# loop over each layer (index i) and report inactive batch (index j) that satisfies
+# convergence criteria for the selected method (window,half, or all)
+for i in range(len(n_layers)):
+  nl = n_layers[i]
+  if(method == "window"):
+      # detect when entropy of current batch is within window (mean +/- std of window)
+      for j in range(window_length,len(entropy[i])):
+          window = entropy[i][(j-window_length):j]
+          window_mean = np.average(window)
+          window_dev = np.std(window)
+          window_low = window_mean - window_dev
+          window_high = window_mean + window_dev
+          # print("window bounds: ", window_low , " to ", window_high )
+          if(window_high - entropy[0][j] > 0 and entropy[0][j]-window_low > 0):
+              # if entropy[j] is less than window_high and greater than
+              # window_low, then it is within the window and we've succeeded
+              print("For layer", nl, ", batch", j, "produced a value within one "
+                    "standard deviation of the window mean. It is recommended to do "
+                    "at least this many inactive cycles for this layer.")
+              break
+          else:
+              continue
+  elif(method == "half"):
+      #  Accordinig to Brown, the below is done in MCNP5:
```

Comment:
Can you put a specific reference here, in case someone wants to cite this MCNP method?